iFixit/react-commerce
# Product Page: Add Nokia partner badge SVG

Closes https://github.com/iFixit/ifixit/issues/46860

### QA

View a product that has the oem_partnership metafield set to `nokia`, and verify that the badge has the same UI as the existing oem partnership badges. You can modify the oem partnership metafield of a product in Shopify Admin.
